news/2022-10-04-changes-to-osu-mania-sr-and-pp.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "Changes to osu!mania Star Rating & Performance Points"
+date: 2022-10-04 12:00:00 +0000
+---
+
+Big changes are arriving in the osu!mania scene as difficulty calculation receives its first set of updates in a long time! Read on for more info!
```

Comment:
i've reworded this based on the original entry line, but are these changes really *big*? it's basically a set of bug fixes with LN and chord calculations plus a pivot from score into accuracy if I'm reading it right

Reply:
The biggest of changes is the Score -> Accuracy change.

Even if they aren't *big* per se, the community sees it differently given how scarce these are to begin with. Should be fine.